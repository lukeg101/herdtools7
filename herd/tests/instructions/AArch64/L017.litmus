AArch64 L017

(* Tests STR barrel shifters *)
(* Same as M002 but using vectors so it works in non-mixed mode*)

{ uint32_t x[2] = {2,2}; 0:x0=1; 0:x2=x; int64_t 0:x3=1;}
P0                        ;
   STR W0, [X2, X3, LSL#2];

exists (x={2,1})
