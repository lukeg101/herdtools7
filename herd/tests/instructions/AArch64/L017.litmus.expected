Test L017 Allowed
States 1
x={2,1};
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x={2,1})
Observation L017 Always 1 0
Hash=4d6e36211c150aee5ee2723464a442f6

